techhive/routes/messages.js

```diff
@@ -0,0 +1,10 @@
+import {addMessage,  getMessages} from '../controllers/messageController.js'
+import express from 'express'
+
+const router = express.Router();
+
+router.post("/addmsg/", addMessage);
+router.post("/getmsg/", getMessages);
```

Comment:
can we use `router.get()` for fetching messages (since we're not reading, not writing data)?